### ISSUE TITLE
Conda init required after conda installation

### ISSUE THREAD

**Issue by Bastiaanspanjaard** [bihealth/bih-cluster#103]
Hello,

As a first-time BIH cluster user, I was following the tutorial how to install conda (https://bihealth.github.io/bih-cluster/best-practice/software-installation-with-conda/). I set up a new environment and tried to activate (`conda activate <environment_name>`) but got an error message:                                                                                                            
```
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.                    
To initialize your shell, run                                                                                 
                                                                                                              
    $ conda init <SHELL_NAME>                                                                                 
                                                                                                              
Currently supported shells are:                                                                               
  - bash                                                                                                      
  - fish                                                                                                      
  - tcsh                                                                                                      
  - xonsh                                                                                                     
  - zsh                                                                                                       
  - powershell                                                                                                
                                                                                                              
See 'conda init --help' for more information and options.                                                     
                                                                                                              
IMPORTANT: You may need to close and restart your shell after running 'conda init'.                           

```                                                                                                              

Indeed, running `conda init`, closing and restarting a shell fixed this and I can now activate the conda environment. However, this may be something to add to the software installation section?

**Comment by messersc** [comment]
Hi Bastiaan,

I would happily accept a merge request with your proposed change!

Best,
clemens

**Comment by sellth** [comment]
fixed via #128
